server/hedley/modules/custom/hedley_migrate/handlers/user/HedleyMigrateUsers.php

```diff
@@ -56,4 +65,59 @@ public function __construct($arguments) {
     );
   }
 
+  /**
+   * Prepare rows before passing the data to the entity creation.
+   *
+   * Add the role id to the user from the role name and keep the name for the
+   * OG role.
+   *
+   * @param object $row
+   *   The row data.
+   *
+   * @return bool
+   *   Success.
+   */
+  public function prepareRow($row) {
+    // User role.
+    $role = user_role_load_by_name($row->role);
+    if (!$role) {
+      return TRUE;
+    }
+
+    // Keep the role's name for OG.
+    $row->og_role = $row->role;
+    $row->role = $role->rid;
+    return TRUE;
+  }
+
+  /**
+   * Grant the appropriate OG roles to each user.
+   *
+   * @param object $entity
+   *   The user entity object that was created.
+   * @param object $row
+   *   The CSV row we are manipulating.
+   */
+  public function complete($entity, $row) {
+    if (!isset($row->og_role)) {
+      // No OG role granted for the user.
+      return;
+    }
+
+    $user_wrapper = entity_metadata_wrapper('user', $entity);
+    $group = reset($user_wrapper->og_user_node->value());
+    if (!$group) {
+      // No role granting required if the user doesn't have a group.
+      return;
+    }
+    $og_roles = og_roles('node', 'municipality', $group->nid);
+    $rid = array_search($row->og_role, $og_roles);
+    og_role_grant('node', $group->nid, $user_wrapper->getIdentifier(), $rid);
+
+    // Add active status for migrated members.
+    $og_membership = og_get_membership('node', $group->nid, 'user', $entity->uid);
```

Comment:
It does but it's `Pending` by default which is good, but not for the migrated users.